backend/app/cmd/cmd.go

```diff
@@ -22,6 +22,7 @@ type CommonOptionsCommander interface {
 	SetCommon(commonOpts CommonOpts)
 	Execute(args []string) error
 	HandleDeprecatedFlags() []DeprecatedFlag
+	FindDeprecatedFlagsCollisions() []DeprecatedFlag
```

Comment:
unlike others, this one has no default implementation in CommonOpts. I guess it may break other commands